plugin/keyring_vault/vault_curl.cc

```diff
@@ -0,0 +1,177 @@
+#include <algorithm>
+#include "vault_curl.h"
+#include "base64.h"
+
+namespace keyring
+{
+
+static size_t write_response_memory(void *contents, size_t size, size_t nmemb, void *userp)
```

Comment:
I think I can limit it to 32MB - this is the size limit that Vault is imposing on request. The response limit can be a problem when retrieving the list of keys from Vault. It is done only once on plugin initialization. With 32 MB limit the number of keys keyring_vault will be able to work with (say 30 bytes for key signature) is around 1milion. Seems like a high number, but still we impose a limit. @laurynas-biveinis what do you think about such limit ?